app/models/provider.rb

```diff
@@ -310,4 +314,8 @@ def services
       )
     end
   end
+
+  def set_provider_name_search
+    self.provider_name_search = QueryNormalizerService.call(query: self.provider_name)
```

Comment:
does it need a token test... ie, mock was called etc?

Reply:
I agree... I think we might be better with a mocked test here as the `QueryNormalizerService` is tested already. We just need to make sure it is used.